### PR TITLE
SCM v6 support via config option

### PR DESCRIPTION
**Pull request by zevanty**
I saw the comments in #130 and decided to implement the config option, since we are using SCM v6 and was running into problems until we saw @gpflaum comments.

The v6 code is a direct copy of @veon82, so all credits go to him. I just made the the config option to speed things up.

### CODE REVIEW

**Comment by WtfJoke** [comment]
Can you try to rebase the pull request on his pull request? So the corresponding changes/credits goes to him. :)

File: config.ini.sample [hunk 1]
```diff
@@ -13,6 +13,8 @@ ScmCommand = lscm
 # Optional - Set encoding of files (For example encoding = UTF-8)
 # See "https://github.com/rtcTo/rtc2git/wiki/Encoding" for further instructions
 encoding =
+# Optional - SCM version. Different SCM versions have different commands. If not specific, default is 5.
+ScmVersion = 5
```
Comment (WtfJoke) [inline config.ini.sample:17]: Could you rename ScmVersion to RTCVersion

**Comment by WtfJoke** [comment]
I merged config changes. I ignored changes on rtcFunctions. This will be done by #130

**Comment by zevanty** [comment]
Sorry, didn't see your comment (was out on an internet-less vacation). I'm fine with your changes. Thanks for the merge.

**Pull request merged**